pages.es/linux/torify.md

```diff
@@ -0,0 +1,32 @@
+# torify
+
+> Enruta el tráfico de red a través de la red Tor.
+> Más información: <https://manned.org/man/torify>.
+
+- Enruta el tráfico a través de Tor:
+
+`torify {{comando}}`
+
+- Activa Tor en el intérprete de comandos:
```

Comment:
```suggestion
- Activa o desactiva Tor en el intérprete de comandos:
```

I am not 100% sure of this suggestion, however, I believe the description should be changed because `activa` does not encompass `toggle` (i.e. on and off). Do you think we should stick to this suggestion or use another word (e.g. `alterna`) @kant?

Reply:
Alterna is ok

Reply:
Hmmm, `alterna` does not sound right for me. Perhaps, we should ask another Spanish-speaking maintainer, may I?

Reply:
@tricantivu Alterna, cambia, intercambia, troca, reemplaza, releva (...)

Reply:
None of those words seem adequate, for the moment, shall we stick to the original suggestion?